proposals/2019/Ensure adequate containerization of lists.txt

```diff
@@ -0,0 +1,179 @@
+<xs:element name="LinkedSystemID" minOccurs="0" maxOccurs="unbounded"> <!-- //element(*,auc:PumpSystemType)/auc:LinkedSystemID -->
+<xs:element name="LinkedSystemID" minOccurs="0" maxOccurs="unbounded"> <!-- //element(*,auc:FanSystemType)/auc:LinkedSystemID -->
+<xs:element name="LinkedSystemID" minOccurs="0" maxOccurs="unbounded"> <!-- //element(*,auc:MotorSystemType)/auc:LinkedSystemID -->
+<xs:element name="CeilingInsulation" minOccurs="0" maxOccurs="unbounded">
+<xs:element name="RoofInsulation" minOccurs="0" maxOccurs="unbounded">
+<xs:element name="GroundCoupling" minOccurs="0" maxOccurs="unbounded">
+<xs:element name="EnergyUseByEndUse" minOccurs="0" maxOccurs="unbounded">
+<xs:element name="UseType" minOccurs="0" maxOccurs="unbounded">
+<xs:element name="RatePeriod" minOccurs="0" maxOccurs="unbounded"> <!-- //element(*,auc:UtilityType)/auc:RateSchedule/auc:TypeOfRateStructure/auc:FlatRate/auc:RatePeriod -->
+<xs:element name="RatePeriod" minOccurs="0" maxOccurs="unbounded"> <!-- //element(*,auc:UtilityType)/auc:RateSchedule/auc:TypeOfRateStructure/auc:TimeOfUseRate/auc:RatePeriod -->
+<xs:element name="RatePeriod" minOccurs="0" maxOccurs="unbounded"> <!-- //element(*,auc:UtilityType)/auc:RateSchedule/auc:TypeOfRateStructure/auc:TieredRate/auc:RatePeriod -->
+<xs:element name="HeatingPlant" type="auc:HeatingPlantType" minOccurs="0" maxOccurs="unbounded">
+<xs:element name="CoolingPlant" type="auc:CoolingPlantType" minOccurs="0" maxOccurs="unbounded">
+<xs:element name="CondenserPlant" type="auc:CondenserPlantType" minOccurs="0" maxOccurs="unbounded">
+<xs:element name="LinkedScheduleID" minOccurs="0" maxOccurs="unbounded"> <!-- auc:LinkedPremises/auc:Site/auc:LinkedSiteID/auc:LinkedScheduleID -->
+<xs:element name="LinkedScheduleID" minOccurs="0" maxOccurs="unbounded"> <!-- auc:LinkedPremises/auc:Facility/auc:LinkedFacilityID/auc:LinkedScheduleID -->
+<xs:element name="LinkedScheduleID" minOccurs="0" maxOccurs="unbounded"> <!-- auc:LinkedPremises/auc:Subsection/auc:LinkedSubsectionID/auc:LinkedScheduleID -->
+<xs:element name="LinkedScheduleID" minOccurs="0" maxOccurs="unbounded"> <!-- auc:LinkedPremises/auc:ThermalZone/auc:LinkedThermalZoneID/auc:LinkedScheduleID -->
+<xs:element name="LinkedScheduleID" minOccurs="0" maxOccurs="unbounded"> <!-- auc:LinkedPremises/auc:Space/auc:LinkedSpaceID/auc:LinkedScheduleID -->
+```
+
+
+List 2: elements that are defined outside of the main schema path (at the bottom of the code) and are referenced throughout the schema code. For mor information, reference the following pull request comment: https://github.com/BuildingSync/schema/pull/67#issuecomment-456912360. 
+
+```xml
+<xs:element name="TenantID">
+<xs:element name="EnergyUseByFuelType">
+```
+
+
+List 3: unique cases to be reviewed by the BuildingSync technical lead to define approach.
```

Comment:
What about making this look like the following?

![image](https://user-images.githubusercontent.com/1907354/51917943-db718b80-239d-11e9-95b2-9efc74fbb1a1.png)